pkgdiff

```diff
@@ -29,7 +29,7 @@ cat2=$(get_category "${2}")
 fn1=${1##*/}
 fn2=${2##*/}
 
-export PORTAGE_TMPDIR=/tmp/pkgdiff
+export PORTAGE_TMPDIR=$(mktemp --tmpdir -d pkgdiff.XXXXXX)
```

Comment:
I don't like this. Right now we're reusing the same directory, so it's easy to inspect stuff manually and clean it up afterwards.

Reply:
Meh. Alright. I think I wanted this for some reason but I don't remember what right now. We can revisit if I remember.